storage/pruning/pruningStorer.go

```diff
@@ -88,6 +88,9 @@ func initPruningStorer(
 	if check.IfNil(args.PathManager) {
 		return nil, storage.ErrNilPathManager
 	}
+	if args.MaxBatchSize > int(args.CacheConf.Size) {
```

Comment:
why not moving this on the persister constructors (SerialDB/DB)?

Reply:
Done